src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -23,11 +23,16 @@
 from bot.fuzzers import engine
 from bot.fuzzers import utils as fuzzer_utils
 from bot.fuzzers.syzkaller import config
+from google_cloud_utils import gsutil
 from metrics import logs
 from system import environment
 from system import new_process
 
 REPRODUCE_REGEX = re.compile(r'reproduced (\d+) crashes')
+CORPUS_BUCKET = environment.get_value('CORPUS_BUCKET')
+GCS_PATH = 'corpus'
+CORPUS_FILENAME = 'corpus.db'
```

Comment:
I think Max spector said to include device family in corpus filename so that when we have multiple bots, we don't override corpus across familieis.

Reply:
Done.